app/models/goal.py

```diff
@@ -3,3 +3,11 @@
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    tasks = db.relationship("Task", back_populates="goal")
+
+    def to_dict(self):
+        goal_as_dict = {}
+        goal_as_dict["id"] = self.goal_id
+        goal_as_dict["title"] = self.title
+        return goal_as_dict
```

Comment:
To DRY up our code and reduce the number of times we have to write `goal_as_dict` we can return the dictionary literal. 

```suggestion
    def to_dict(self):
        return {
            "id": self.goal_id,
            "title": self.title
        }
```